.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing to Up for Grabs
+
+This guide will help you get setup so you can make changes to the project, view
+them locally and verify that the site will deploy correctly when opening a pull
+request.
+
+## Project Setup
+
+This repository contains the GitHub Pages content and some additional Ruby and
+NodeJS tooling to help verify changes as part of publishing the site.
+
+To get a local clone of the repository:
+
+```
+git clone https://github.com/up-for-grabs/up-for-grabs.net.git
+```
+
+If you have a fork of the repository on GitHub, replace the first `up-for-grabs`
+with your GitHub account.
+
+We require Ruby 2.6, Bundler 2 and Node 10+ to test the site. You can check
+these are present by running these commands in a terminal:
+
+```
+$ ruby -v
+ruby 2.6.4p104 (2019-08-28 revision 67798) [x86_64-darwin18]
+$ bundle -v
+Bundler version 2.0.2
+$ node -v
+v10.16.3
+```
+
+## Testing the site
+
+There are two recommended ways to test the site locally, based on what OS you
+are currently using:
+
+- for macOS or Linux contributors - either approach is fine
+- for Windows contributors - Docker is recommended because Ruby on Windows has
+  some issues that need experience to properly workaround
```

Comment:
need `some` experience will be better

Reply:
I ended up going in a slightly different direction here because it felt that mentioning `some` twice in close succession felt excessive:

> Docker is recommended because Ruby on Windows has some issues that need familiarity with the toolchain to properly workaround

Reply:
Ohk fine